spec/factories/gias/school_factory.rb

```diff
@@ -10,7 +10,7 @@
     establishment_number { Faker::Number.unique.within(range: 1..9_999) }
     phase_name { "Phase one" }
     urn { Faker::Number.unique.within(range: 10_000..9_999_999) }
-    ukprn { Faker::Number.unique.number(digits: 5) }
+    ukprn { rand(10_000_000..99_999_999).to_s }
```

Comment:
Were we getting duplicates with how it was previously? I wonder if a sequence would be more reliable? 🤔